gsa/src/stories/radio.js

```diff
@@ -0,0 +1,108 @@
+/* Copyright (C) 2019 Greenbone Networks GmbH
+ *
+ * SPDX-License-Identifier: GPL-2.0-or-later
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+
+import React from 'react';
+import {storiesOf} from '@storybook/react';
+import RadioComponent from '../web/components/form/radio';
+
+class TestRadio extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      notification: '',
+      value: '',
+    };
+    this.handleChange = this.handleChange.bind(this);
+  }
+
+  handleChange(value, name) {
+    if (value === '1') {
+      this.setState(state => ({
+        value: 'coffee',
+      }));
+    } else if (value === '2') {
+      this.setState(state => ({
+        value: 'tea',
+      }));
+    } else if (value === '3') {
+      this.setState(state => ({
+        value: 'water',
+      }));
+    }
+  }
+
+  render() {
+    const text = 'You chose ' + this.state.value;
+    return (
+      <div>
+        <ul>
+          <li>
+            Coffee
+            <RadioComponent
+              name="radio1"
+              value="1"
```

Comment:
You could use the expected values directly here like
```suggestion
              value="coffee"
```